docs/chapters/sdk/run-binary-on-target.rst

```diff
@@ -0,0 +1,21 @@
+Deploying and running binary on target
+======================================
+
+Once the binary has been cross compiled it needs to be installed on the target. To do so the easiest way is to use SSH. For that to work you need to know the IP address of your target.
+
+.. note:: The `Raspberry Pi documentation`_ describes how to find a new device on the network. Finding the IP address for our device will work in the same manner.
+
+.. _`Raspberry Pi documentation`: https://www.raspberrypi.org/documentation/remote-access/ip-address.md
+
+The default user and at the same time password on the PELUX image is `root`. Use `scp` to copy your binary to the target.
+
+.. code-block:: bash
+
+    scp build/template-service root@192.168.1.2:
```

Comment:
It's just a example IP address. But perhaps I shouldn't use it and instead write something like `ssh root@<ip address>`.